libnvidia-container.patch

```diff
@@ -0,0 +1,120 @@
+diff --git a/src/ldcache.c b/src/ldcache.c
+index 38bab05..ba922d9 100644
```

Comment:
Could you add one or two lines at the top explaining what the patch does and where it comes from?